NuGet.Config

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+
+<!--
+    This configuration tells the .NET Core SDK to build the Q# runtime using the
```

Comment:
This really just tells .NET Core how to find prereleased packages. 
Each project then tells .NET Core which version of the package they want to use, and most of the projects in `master` use prereleased versions, which is why it's needed. I would tailor the comment to reflect this.